core/README.md

```diff
@@ -278,7 +306,7 @@ present each of these as a catalog, and create parent catalogs for them that all
       - child -> /catalogs/sentinel_5p_cloud
   - child -> /catalogs/modis
     - child -> /catalogs/modis_mcd43a4
-    - child -> /catalogs/modis_mod11a1 
```

Comment:
The mermaid stuff above is fancy. Maybe we should also use it for these examples?

Reply:
Filed https://github.com/radiantearth/stac-api-spec/issues/386 for that